.buildkite/scripts/steps/functional/security_serverless_rule_management_prebuilt_rules.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/steps/functional/common.sh
+source .buildkite/scripts/steps/functional/common_cypress.sh
+
+export JOB=kibana-security-solution-chrome
+export KIBANA_INSTALL_DIR=${KIBANA_BUILD_LOCATION}
+
+echo "--- Investigations Cypress Tests on Serverless"
```

Comment:
This looks like it needs updating

Reply:
True!! I bet you cannot figure out which was the script I initially copied 🫣